### ISSUE TITLE
slabs in 3D spherical model

### ISSUE THREAD

**Issue by ZonglinGuo** [GeodynamicWorldBuilder/WorldBuilder#741]
![slab_anomaly](https://github.com/GeodynamicWorldBuilder/WorldBuilder/assets/161077546/8640f3be-5524-4812-9071-bfc4fc66a454)
Hi, in my spherical 3D model, I intented to set a trench form (30,-15) to (30,15). When I run "gwbgrid this.wb 3D.grid"  to do a test, the result is perfect, however there are some anomaly for slabs when I run in supercomputer, as shown in attached file and code. Any suggestion?
    {
      "model":"oceanic plate", "name":"Subducting Oceanic plate", "max depth":100e3, 
      "coordinates":[[-30,-15],[30,-15],[30,15],[-30,15]],

       "composition models":[{"model":"uniform", "compositions":[0], "max depth":100e3}]
    },
    { 
      "model":"subducting plate", "name":"Slab2", "dip point":[50,0],
      "coordinates":[[30,-15],[30,15]],
      "segments":
        [
          {"length":300e3, "thickness":[100e3],  "angle":[0,60],
           "composition models":[
            {"model":"uniform", "compositions":[0], "max distance slab top":100e3}]},
          {"length":600e3, "thickness":[100e3], "angle":[60],
           "composition models":[
            {"model":"uniform", "compositions":[0], "max distance slab top":100e3}]}
        ]
    }

**Comment by MFraters** [comment]
Thanks for reporting the issue. Can you post the whole world builder file (.wb) and your grid file (.grid)? Thanks!

**Comment by ZonglinGuo** [comment]
Yes, this is my .wb file,both of two slabs are tricky as shown in my previous figure.
{
  "version": "0.6",
  "coordinate system":{"model":"spherical", "depth method":"begin segment", "radius":1.0},
  "features":
  [

  
    {
      "model":"oceanic plate", "name":"Side plate1", "max depth":100e3, 
      "coordinates":[[-180,-15],[180,-15],[180,-90],[-180,-90]],

       "composition models":[{"model":"uniform", "compositions":[1], "max depth":100e3}]
    },

    {
      "model":"oceanic plate", "name":"Side plate2", "max depth":100e3, 
      "coordinates":[[-180,15],[180,15],[180,90],[-180,90]],

       "composition models":[{"model":"uniform", "compositions":[1], "max depth":100e3}]
    },   

    {
      "model":"oceanic plate", "name":"Side plate3", "max depth":100e3, 
      "coordinates":[[-180,-15],[-30,-15],[-30,15],[-180,15]],

       "composition models":[{"model":"uniform", "compositions":[1], "max depth":100e3}]
    },

    {
      "model":"oceanic plate", "name":"OP", "max depth":100e3, 
      "coordinates":[[30,-15],[90,-15],[90,15],[30,15]],

       "composition models":[{"model":"uniform", "compositions":[2], "max depth":100e3}]
    },
    
    {
      "model":"oceanic plate", "name":"Side plate4", "max depth":100e3, 
      "coordinates":[[90,-15],[180,-15],[180,15],[90,15]],

       "composition models":[{"model":"uniform", "compositions":[1], "max depth":100e3}]
    },   
        
    {
      "model":"oceanic plate", "name":"Subducting Oceanic plate", "max depth":100e3, 
      "coordinates":[[-30,-15],[30,-15],[30,15],[-30,15]],

       "composition models":[{"model":"uniform", "compositions":[0], "max depth":100e3}]
    },
    
       
    { 
      "model":"subducting plate", "name":"Slab", "dip point":[50,0],
      "coordinates":[[10,-15],[20,15]],
      "segments":
        [
          {"length":300e3, "thickness":[100e3],  "angle":[0,60],
           "composition models":[
            {"model":"uniform", "compositions":[0], "max distance slab top":100e3}]},
          {"length":600e3, "thickness":[100e3], "angle":[60],
           "composition models":[
            {"model":"uniform", "compositions":[0], "max distance slab top":100e3}]}
        ]
    },
    
    { 
      "model":"subducting plate", "name":"Slab2", "dip point":[50,0],
      "coordinates":[[30,-15],[30,15]],
      "segments":
        [
          {"length":300e3, "thickness":[100e3],  "angle":[0,60],
           "composition models":[
            {"model":"uniform", "compositions":[0], "max distance slab top":100e3}]},
          {"length":600e3, "thickness":[100e3], "angle":[60],
           "composition models":[
            {"model":"uniform", "compositions":[0], "max distance slab top":100e3}]}
        ]
    }

  ]
}

    


Here is 3D grid file:


grid_type = chunk
dim = 3
compositions = 4

# domain of the grid
x_min = -180
x_max = 180 
y_min = -90
y_max = 90
z_min = 5771e3 
z_max = 6371e3 

# low res grid properties
n_cell_x = 720 
n_cell_y = 360
n_cell_z = 40

**Comment by MFraters** [comment]
I can get the model throught the gwb-grid which you showed in the top image. Is the bottom image run through aspect? If so, can you provide the top part of the log.txt file and if possible the .prm file?
